pkg/printer/printer_test.go

```diff
@@ -0,0 +1,7 @@
+package printer
+
+import "testing"
+
+func Test_interface(t *testing.T) {
+	var _ Interface = &Printer{}
```

Comment:
Do we really need a test for this? It can be checked at compile time if it's at the `printer` package level

Reply:
It's tested at compile time. If `*Printer` doesn't fulfil the `Interface` then this code won't compile. That's a standard practice.

Reply:
What I meant was: why does it have to be in a test suite if it is ran at compile time?

Reply:
It isn't checked in this codebase otherwise. Should I remove it?

Reply:
Judging by what I've seen so far in Go codebases, the line of code would be better suited in `printer.go`, alongside the `Printer` struct definition, not in a test plan

Reply:
With you. Can you make a suggestion? I'm happy to have that line somewhere else :D